microdnf/utils.cpp

```diff
@@ -19,61 +19,19 @@ along with microdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include "utils.hpp"
 
-#include <fcntl.h>
-#include <libsmartcols/libsmartcols.h>
-#include <pwd.h>
-#include <sys/stat.h>
-#include <unistd.h>
-
-#include <cstdlib>
-#include <cstring>
-#include <iostream>
-#include <stdexcept>
+#include <libdnf/common/proc.hpp>
```

Comment:
Add `#include <unistd.h>`. For `getuid` and `getpid`.

Reply:
fixed